### PR TITLE
implementation bussines card

### PR DESCRIPTION
**Pull request by briansetzesglobant**
I did the implementation of the second exercise

image

Icon of the application
icono

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -5,109 +5,74 @@ void main() {
...
-        child: Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
-    );
+        home: Scaffold(
+          backgroundColor: Color.fromARGB(255, 0, 217, 255),
+          body: SafeArea(
+            top: true,
+            bottom: true,
+            left: true,
+            right: true,
```
Comment (MarcosCGlobant) [inline lib/main.dart]: this values of SafeArea are true by default, you can omit them

File: lib/main.dart [hunk 2]
```diff
@@ -5,109 +5,74 @@ void main() {
...
         ),
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
-    );
+        home: Scaffold(
+          backgroundColor: Color.fromARGB(255, 0, 217, 255),
```
Comment (MarcosCGlobant) [inline lib/main.dart]: Your are missing a trailing comma before the closing parenthesis

File: lib/main.dart [hunk 3]
```diff
@@ -5,109 +5,74 @@ void main() {
...
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  CircleAvatar(
+                    backgroundImage: AssetImage(
+                      'assets/images/avatar.jpg',
+                    ),
+                    radius: _radius,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.all(5),
```
Comment (MarcosCGlobant) [inline lib/main.dart]: Your are missing a trailing comma before the closing parenthesis

File: lib/main.dart [hunk 4]
```diff
@@ -5,109 +5,74 @@ void main() {
...
+                      style: TextStyle(
+                        backgroundColor: Colors.lightGreenAccent,
+                        fontSize: 25.0,
+                        letterSpacing: 3.0,
+                        color: Colors.blue,
+                      ),
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.all(5),
```
Comment (MarcosCGlobant) [inline lib/main.dart]: Your are missing a trailing comma before the closing parenthesis

File: lib/main.dart [hunk 5]
```diff
@@ -5,109 +5,74 @@ void main() {
...
+        home: Scaffold(
+          backgroundColor: Color.fromARGB(255, 0, 217, 255),
+          body: SafeArea(
+            top: true,
+            bottom: true,
+            left: true,
+            right: true,
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
```
Comment (MarcosCGlobant) [inline lib/main.dart]: Another approach to this is removing this Center wrapping, and adding CrossAxisAlignment.center on the column, you will get the same effect

File: lib/main.dart (hunk 1 above)
Comment (briansetzesglobant) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 2 above)
Comment (briansetzesglobant) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 3 above)
Comment (briansetzesglobant) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 4 above)
Comment (briansetzesglobant) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 5 above)
Comment (briansetzesglobant) [reply lib/main.dart]: Done

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**